test/DriverFactoryTest.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * @license See the file LICENSE for copying permission
+ */
+
+namespace ContainerInteropDoctrineTest;
+
+use OutOfBoundsException;
+use ContainerInteropDoctrine\DriverFactory;
+use Interop\Container\ContainerInterface;
+use PHPUnit_Framework_TestCase as TestCase;
+
+class DriverFactoryTest extends TestCase
+{
+    public function testMissingClassKeyWillReturnOutOfBoundException()
+    {
+        $container = $this->prophesize(ContainerInterface::class);
+        $factory = new DriverFactory();
+
+        $this->setExpectedException(OutOfBoundsException::class, 'Missing "class" config key');
```

Comment:
`setExpectedException` is deprecated, should use `expectException` and `expectExceptionMessage`